kasa/module.py

```diff
@@ -132,6 +137,44 @@ def __init__(self, device: Device, module: str):
         self._device = device
         self._module = module
         self._module_features: dict[str, Feature] = {}
+        self._bound_feature_ids: dict[property, str] = {}
+
+    def _bound_feature_id(self, attr: property) -> str | None:
+        """Get bound feature for module or none if not supported."""
+        if attr in self._bound_feature_ids:
+            return self._bound_feature_ids[attr]
+        if isinstance(attr, property):
+            hints = get_type_hints(attr.fget, include_extras=True)
+            if (return_hints := hints.get("return")) and hasattr(
+                return_hints, "__metadata__"
+            ):
+                metadata = hints["return"].__metadata__
+                for meta in metadata:
+                    if isinstance(meta, FeatureProperty):
+                        self._bound_feature_ids[attr] = meta.id
+                        return meta.id
+        return None
+
+    def is_bound_feature(self, attr: property) -> bool:
+        """Return True if property is bound to a feature."""
+        return bool(self._bound_feature_id(attr))
+
+    def has_bound_feature(self, attr: property) -> bool:
+        """Return True if the bound property feature is supported."""
+        if id := self._bound_feature_id(attr):
+            return id in self._module_features
+        raise KasaException("")
```

Comment:
Do we really need so much new public API? Wouldn't it be enough to have the `get_bound_feature` and let consumers do something like:
```
if feat := module.get_bound_feature(property):
    feat.foobar()
```
?

Reply:
I didn't see the harm in exposing these especially as it's not that intuitive to have to `get_feature` simply to test whether it supports a feature before calling something `light.hsv`.

Reply:
`is_bound_feature` now removed